src/components/TitleLogo/TitleLogo.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./styles.scss";
+
+export const TitleLogo = () => {
+  return (
+    <div className="title__row">
```

Comment:
why is it "title__row" instead of "title_row"?

same for the rest of className

Reply:
We will be following the [BEM](http://getbem.com/naming/) methodology